src/components/common/DropdownComponent.jsx

```diff
@@ -25,7 +25,7 @@ DropdownComponent.propTypes = {
   options: PropTypes.array.isRequired,
   placeHolder: PropTypes.string,
   onChange: PropTypes.func,
-  name: PropTypes.string.isRequired,
+  name: PropTypes.string,
```

Comment:
This change won't be necessary once Thuku merges his PR as he has fixed it. Please pester him to merge it ASAP 🙂

Reply:
😄okay